### ISSUE TITLE
When deploying a VDC, the user should be informed that this might take several minutes.

### ISSUE THREAD

**Issue by ken-de-moor** [threefoldtech/js-sdk#2390]
I'm currently on a very fast PC with good specs and good internet. As a user I am not expecting anything to take more than a few seconds. If a site does have to do actions in the background I expect there to be a specific warning to tell me about having to wait.

1) While the deploy VDC is loading, we should have a text show right under the loading bar that informs the user, something like "Your VDC is being deployed, this might take several minutes"

**Comment by rkhamis** [comment]
already vdc creation says it take time. I don't see when else this should be added.

**Comment by xmonader** [comment]
@ken-de-moor where do you suggest this be added? as @rkhamis said, it's already in the VDC creation?

**Comment by ken-de-moor** [comment]
Sorry haven't had time in a while to check back on VDC UX stuff.

@xmonader @rkhamis 

I have not seen any message about this, but my original bug report was not clear enough in what step of the flow I was refering to, my bad.

Step 1) I log in with my ThreeFold Connect

Step 2) I chose a VDC name and secret, and plan

=> Here is the long load that I don't expect. I see a loading bar that loads for quite a while without anything noticeable changing, first time using the VDC I already refreshed this page because I didn't realize it was just taking a while. 

Step 3) Payment .. and after this payment there are messages about slow load.
